### PR TITLE
Update 01-streaming-from-kafka.ipynb

### PR DESCRIPTION
**Pull request by petermarshallio**
Proposed changes to this notebook to apply the revised template for notebooks, simplify some areas (e.g. switch to auto schema ingestion), and give a light introduction to what ingestion specs are.

### CODE REVIEW

**Comment by petermarshallio** [comment]
@techdocsmith tagging you as I believe this was one of your contributions?

File: notebooks/02-ingestion/01-streaming-from-kafka.ipynb [hunk 1]
```diff
@@ -328,17 +340,17 @@
...
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this social media scenario, each incoming event represents a post on social media, for which you collect the timestamp, username, and post metadata. You are interested in analyzing the total number of upvotes for all posts, compared between users. Preview this data with the following query:"
+    "In this social media scenario, each incoming event represents a post on social media, for which you collect the timestamp, username, and post metadata. You are interested in analyzing the total number of upvotes for all posts, compared between users.\n",
+    "\n",
+    "Run this next cell to execute the query, store the results (`response`), and display them."
```
Comment (techdocsmith) [inline notebooks/02-ingestion/01-streaming-from-kafka.ipynb:353]:
```suggestion
    "Run the next cell to execute the query, store the results (`response`), and display them."
```

File: notebooks/02-ingestion/01-streaming-from-kafka.ipynb [hunk 2]
```diff
@@ -365,7 +377,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Visualize the total number of upvotes per user using a line plot. You sort the results by username before plotting because the order of users may vary as new results arrive."
+    "Visualize the total number of upvotes per user using a line plot.\n",
+    "\n",
+    "In the next cell, the results are put into a Pandas dataframe and then sorted (the order of users may vary as new results arrive).\n",
+    "\n",
+    "Next, the dataframe is plotted."
```
Comment (techdocsmith) [inline notebooks/02-ingestion/01-streaming-from-kafka.ipynb:384]:
```suggestion
    "Next, plot the the dataframe."
```

File: notebooks/02-ingestion/01-streaming-from-kafka.ipynb [hunk 3]
```diff
@@ -365,7 +377,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Visualize the total number of upvotes per user using a line plot. You sort the results by username before plotting because the order of users may vary as new results arrive."
+    "Visualize the total number of upvotes per user using a line plot.\n",
+    "\n",
+    "In the next cell, the results are put into a Pandas dataframe and then sorted (the order of users may vary as new results arrive).\n",
```
Comment (techdocsmith) [inline notebooks/02-ingestion/01-streaming-from-kafka.ipynb:382]:
```suggestion
    "The next cell stores the results in a Pandas dataframe and then sorts them. Note that the order of users may vary as new results arrive.\n",
```

**Review by techdocsmith — CHANGES_REQUESTED** [review]
love the improvements. left a heap of stylistic comments.

**Review by techdocsmith — CHANGES_REQUESTED** [review]
love the improvements. left a heap of stylistic comments.

**Pull request merged**